Pod/PagesDemo/AppDelegate.swift

```diff
@@ -8,7 +8,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
 
       let viewController1 = UIViewController()
-      viewController1.view.backgroundColor = UIColor.redColor()
```

Comment:
So, do you want to use `.redColor()` instead of `UIColor.redColor()`
You can but I not sure it you should :)